### ISSUE TITLE
[Bug]: Toast notification is not output by the screen reader

### ISSUE THREAD

**Issue by orsolya-gyovai** [sbb-design-systems/lyne-components#3683]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Accessibility

### It affects the following packages

Components

### Operating system

Windows 11

### Browser / Browser version

Chrome, Firefox, Edge 

### Input mode

None

### What happened?

Dear All,
I need your help: the toast notifications are not output by the screen reader.
I tested it in Chrome, Firefox, Edge with NVDA and JAWS. As far as i remember, the assertive mod was output a few weeks ago. But now nothing at all.
(On android with talkback it works.)
https://lyne-storybook.app.sbb.ch/?path=/story/elements-sbb-toast--with-action-link

Vielleicht sollte das aria-live Attribut an das sbb-toast tag implementiert werden?
Perhaps the `<sbb-toast>` tag should be given the `aria-live` attribute? Could you check please?
Thank You very much!!!

### Steps to reproduce the issue

https://lyne-storybook.app.sbb.ch/?path=/story/elements-sbb-toast--with-action-link
The toast notification should be output by the screen reader.

### Relevant log output

```shell

```
